### ISSUE TITLE
Add Color.ToHSV()

### ISSUE THREAD

**Issue by SaqibS** [xamarin/Essentials#1156]
My core requirement is to have a method:

```
Color.Red.ToHSV(out int hue, out int saturation, out int brightness);
```

Initially, I wouldn't have thought this belongs in Xamarin.Essentials, but I see that there is a `ColorConverters` class, and also `ColorExtensions`, so perhaps this is the right place.

For completeness, I would suggest:

* Add HSV methods equivalent to the existing HSL methods, like `ColorConverters.FromHSV`.
* Then add extension methods `ToHSV()` and `ToHSL()`.

Again, this could easily live outside of Essentials, so interested in comments on this.

**Comment by Redth** [comment]
I think this definitely makes sense to add. Any hints at what the implementation looks like for this?

**Comment by SaqibS** [comment]
As mentioned, the method I personally care the most about is `Color.ToHSV()`. For this, I would take inspiration from the Skia implementation: [here](https://github.com/google/skia/blob/master/src/core/SkColor.cpp) and cross-reference with [this](https://en.wikipedia.org/wiki/HSL_and_HSV) Wikipedia article on HSV/HSL.

**Comment by pictos** [comment]
@Redth can I implement this one?

**Comment by Redth** [comment]
@pictos  please!

**Comment by SaqibS** [comment]
Thanks @pictos and all! :-)

**Comment by jamesmontemagno** [comment]
Merged and will be in 1.6.0
